src/Microsoft.ML.Data/Evaluators/EvaluatorUtils.cs

```diff
@@ -1591,10 +1619,11 @@ private static string GetFoldMetricsAsString(IHostEnvironment env, IDataView dat
             else
                 rowLabelFormat = string.Format("{{1,{0}}} ||", paddingLen);
 
+            var confusionTable = confusionMatrix.ConfusionTableCounts;
```

Comment:
>var confusionTable = confusionMatrix.ConfusionTableCounts; [](start = 12, length = 58)

SO MUCH CONFUSION!
no, seriously, how confusionTable can be  ConfusionTableCounts in  ConfusionMatrix?